pkg/internal/mock/edv/client.go

```diff
@@ -12,12 +12,12 @@ import (
 // Client is the mock edv client
 type Client struct {
 	edvServerURL            string
-	readDocumentReturnValue []byte
+	ReadDocumentReturnValue *operation.EncryptedDocument
 }
 
 // NewMockEDVClient is the mock version of edv client
-func NewMockEDVClient(edvServerURL string, readDocumentReturnValue []byte) *Client {
-	return &Client{edvServerURL: edvServerURL, readDocumentReturnValue: readDocumentReturnValue}
+func NewMockEDVClient(edvServerURL string, readDocumentReturnValue *operation.EncryptedDocument) *Client {
```

Comment:
This is due to a change in the interface defined in the EDV dependency (which was updated in this PR).